src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -39,15 +41,22 @@ const StyledSearch = styled(Input.Search)`
 `;
 
 let map: google.maps.Map;
+const mapId = '76c08a2450c223d9';
```

Comment:
⛏️ Can we move these constants to a different file?

Reply:
Would a new file under utils make sense? Or within the mapPageComponents directory?

Reply:
I would say within mapPageComponents, we can make a tech debt ticket to move any numbers or other random constants for maps into it